packages/onboarding/src/components/woo-payment-gateway-setup/index.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
I know this is how it previously was, but we might just use `index.js` for exports and move the content into it's own file for these.

Reply:
Done.